src/api.js

```diff
@@ -17,3 +26,21 @@ export const getSampleFolder = async () => {
   const body = await response.json();
   return body;
```

Comment:
```suggestion
  return response.json();
```
[중요도: 중간] 으로 하셔도 됩니다~ .json() 이 비동기도 아니기도 하고, 굳이 새로 변수 선언해서 return할 필요도 따로 없어요~

Reply:
수정했습니다. [ceabed4](https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/217/commits/ceabed46da6f5b3026ce4b14fc7ddece383ac82a)